docs/references/magic_link_login.md

```diff
@@ -0,0 +1,57 @@
+# Magic Link Login
+
+Magic Link Login is a feature that allows users to log in if they forget their
+password.
```

Comment:
Or use as a one-time password.

Reply:
Using Magic Link Login as a one-time password authentication?
I don't know if it is supposed to be used for anything other than forgotten passwords.